platform/ios/PolyPodApp/PolyPod/HomeScreenView.swift

```diff
@@ -474,6 +474,7 @@ struct DeveloperSectionView: View {
             VStack(alignment: .leading, spacing: HomeScreenConstants.TileContainer.verticalSpacing) {
                 ForEach(sectionModel.cards) { card in
                     MediumCardView(card: card)
+                        .accessibilityElement()
```

Comment:
@GhenadiePusca After seeing a PR from Tim, it occurred to me that we might need to add this, and that was it! My hypotheses is that stuff like `StaticText` and `Image` already have accessibility elements, but our card not, by default. So without creating one, the accessibility identifier would be applied to all immediate children that have them. Now that I've added this, _only_ the card has it.

Reply:
Yes, that's correct.